lib/jsmidgen.js

```diff
@@ -408,6 +408,15 @@ var Midi = {};
 		return bytes;
 	};
 
+	File.prototype.setTicks = function(num) {
+		// Do nothing in case of invalid input
+		if (!num || typeof num !== 'number') {
+			return;
```

Comment:
Much better to `throw new Error('Invalid input');` than just silently fail, otherwise the user doesn't know that anything has gone wrong.